src/main/java/com/exemple/supermarket/controllers/MainController.java

```diff
@@ -0,0 +1,31 @@
+package com.exemple.supermarket.controllers;
+
+import com.exemple.supermarket.service.SuperMarketService;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.client.HttpClientErrorException;
+
+
+@Controller
+public class MainController {
+
+    @RequestMapping(method = {RequestMethod.POST}, value = "/getProduct")
+    public String sendRequestToStorehouse(@ModelAttribute("name") String name, Model model) throws Exception {
+        ResponseEntity<String> response = SuperMarketService.sendRequestToStorehouse(name);
```

Comment:
Instead of using static method, add SuperMarketService as class variable. Read about dependencie injection to understand how you should use spring in this situation

Reply:
done